db/projects/eos/project.yaml

```diff
@@ -0,0 +1,66 @@
+name: EOS
+web: https://eos.io/
+start_date: "2017-06-26"
+assets:
+  - id: eos
+    type: Token
+    name: EOS
+    symbol: EOS
+    start_date: "2017-07-01"
+    token_properties:
+      platform: Ethereum
+      contract_address: "0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0"
+      decimals: 18
+    ico_properties:
+      start_date: "2017-06-26"
+      end_date: "2018-06-01"
+      total_cap: 900000000
+      raised: 185000000 USD
+    networks:
+      - id: main
+        type: main
+        name: EOS Network
+        proof_type: DPoS
+        mineable: false
+        algorithm: DPoS
```

Comment:
DPoS je sice algoritmus, ale algoritmus consensu coz my mame pod `proof_type` - zkousel jsem googloval a mozna by to mohlo byt "TAPOS", ale radeji bych to asi nechal nevyplnene
`algoritm:` je algoritmus hashovani